pages/urlEncodeDecode.tsx

```diff
@@ -109,21 +116,82 @@ export default function URLEncodeDecode() {
             )}
           </Box>
         </Box>
+
         <TextField
           label='Encoded'
-          data-testid='Encoded'
+          data-testid='Output - Encoded'
           value={encoded}
           onChange={(e) => setEncoded(e.target.value)}
           multiline
         />
+        <Box
+          display='flex'
+          flexWrap='wrap'
+          justifyContent='end'
+          gap={2}
+        >
+          <Button
+            startIcon={<ContentCopyIcon />}
+            disabled={!text}
+            onClick={() => {
+              navigator.clipboard.writeText(encoded || '').then(
+                () => {
+                  setToastMessage('Copied to clipboard');
+                  setToastSeverity('success');
+                  setToastOpen(true);
+                },
+                () => {
+                  setToastMessage('Failed to copy to clipboard');
+                  setToastSeverity('error');
+                  setToastOpen(true);
+                },
+              );
+            }}
+          >
+            Copy
+          </Button>
+        </Box>
         <TextField
           label='Decoded'
-          data-testid='Decoded'
+          data-testid='Output - Decoded'
           value={decoded}
           onChange={(e) => setDecoded(e.target.value)}
           multiline
         />
+        <Box
+          display='flex'
+          flexWrap='wrap'
+          justifyContent='end'
+          gap={2}
```

Comment:
I don't think we need this gap here, it makes the inputs on the page not uniform because the first input doesn't have a gap.